addons/common/init_perFrameHandler.sqf

```diff
@@ -19,6 +19,7 @@ GVAR(waitUntilAndExecArray) = [];
 
 // per frame handler system
 [QFUNC(onFrame), {
+    SCRIPT(onFrame);
```

Comment:
I have wished for this stuff for SOOOO long, to be able to see it in the debugger. Which it can't because there is no identifying factor in there.
Maybe just move it into a SQF file ^^